caffe2/ideep/operators/order_switch_ops.cc

```diff
@@ -0,0 +1,60 @@
+#include <caffe2/ideep/ideep_utils.h>
+
+namespace caffe2 {
+
```

Comment:
If you declare op in .cc file. Please put the op class inside an anonymous namespace to avoid potential link conflict.

Reply:
not sure if correctly understanding your comments.
this op impl is same as others, say, conv, pooling.
All are put in .cc files.
If there's link conflict, perhaps the fallback wrappers of NHWC2NCHW and NCHW2NHWC are still in  caffe2/ideep/operators/operator_fallback_ideep.cc, which should be removed already.

Reply:
Just wrap your local classes in anonymous namespace.